packages/core/docs/vibe-3-changelog.md

```diff
@@ -143,32 +154,40 @@ codemod: `avatar-component-migration`
 
 - `Search` has changed - follow the [`Search` docs](https://style.monday.com/?path=/docs/inputs-search--docs) for the new API.
 
+- ## SearchComponent
+
+codemod: `search-component-import-migration`
+
+- `SearchComponent` component removed - use [`Search`](https://style.monday.com/?path=/docs/inputs-search--docs). [codemod ✅]
+
 ## SplitButton
 
-- We're now accepting instead of static props, inline string, e.g. "bottom-start". We need to change DialogPosition to be a string in its root declaration (this also requires big codemod changes probably) [codemod]
-- Requires codemod for when people used secondaryDialogPosition={SplitButton.secondaryBlaBla}. [codemod]
-- Once changing DialogPosition to be a const, instead of enum, remove DialogPosition double declaration from SplitButton's declaration [internal]
-- Remove `data-testId` prop from nested elements
+- [!!!] We're now accepting instead of static props, inline string, e.g. "bottom-start". We need to change DialogPosition to be a string in its root declaration (this also requires big codemod changes probably) [codemod]
+  - Requires codemod for when people used secondaryDialogPosition={SplitButton.secondaryBlaBla}. [codemod]
+  - Once changing DialogPosition to be a const, instead of enum, remove DialogPosition double declaration from SplitButton's declaration [internal]
+- Remove `data-testId` prop from nested elements (data-testId used to be applied also to each of the internal elements)
 
-### Steps
+### Steps 🚩
```

Comment:
Codemod infra for changing a prop value
https://monday.monday.com/boards/6841644723/pulses/7109083611

Reply:
Codemod for Steps - replace isOnPrimary boolean flag prop to color prop with value of Steps.colors.PRIMARY
https://monday.monday.com/boards/6841644723/pulses/7109110438